.github/workflows/stale-tasks.yml

```diff
@@ -0,0 +1,26 @@
+# This is a workflow that runs periodically to mark issues/prs with no recent activity as stale, it also closes/removes label based on following activity
+
+name: Mark and close stale issues/PRs
+on:
+  schedule:
+  - cron: '0 0 * * *' # runs daily at 12:00 am UTC
+
+jobs:
+  stale:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/stale@v4
+        with:
+          stale-issue-label: stale
```

Comment:
If we remove the stale label, does it restart the `days-before-stale` counter?

Reply:
Any updates to the issue resets the counter. The action uses github isssue updated_at field to calculate when an issue is marked as stale/closed

Reply:
i suspect yes ... the action uses the issue/pr updated_at date which is updated by changes to labels. assignees, comments, etc.